src/options.ts

```diff
@@ -124,8 +123,18 @@ export function optionsFromParameter(parameter: string | undefined): Options {
     }
     Object.assign(options, parsed);
   }
-  // We should promote onlyTypes to its own documented flag, but just an alias for now
-  if (!options.outputJsonMethods && !options.outputEncodeMethods && !options.outputClientImpl && !options.nestJs) {
+  // onlyTypes=true implies outputJsonMethods=false,outputEncodeMethods=false,outputClientImpl=false,nestJs=false
+  if (options.onlyTypes) {
```

Comment:
...huh, yeah, not sure why we weren't doing this before. Great fix!

If you want to add test cases to `options-test.ts` to make sure these don't regress, that'd be great!

Reply:
This should probably have been fixed when the option was documented. :)

I'll add test cases!

Reply:
Tests added.